src/cswinrt/main.cpp

```diff
@@ -321,7 +321,15 @@ Where <spec> is one or more of:
                 group.add([&componentActivatableClasses, &projectionFileWritten]
                 {
                     writer wm;
-                    wm.write("// This file was generated by cswinrt.exe version %", VERSION_STRING);
+                    wm.write(R"(//------------------------------------------------------------------------------
+// <auto-generated>
+//     This file was generated by cswinrt.exe version %
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+)", VERSION_STRING);
```

Comment:
Can you also add a similar write line below to `writer eventHelperWriter("WinRT");`.  It should probably have one, but seems to be have been missed before.

Reply:
Done. Let me know if this is what you meant.

Reply:
Thanks, that is what I meant.